stock_picking_actual_date/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+from . import account_move
+from . import res_currency
+from . import stock_picking
+from . import stock_move
+from . import stock_move_line
```

Comment:
Put files in alphabetical order unless there is reason to do otherwise.
```suggestion
from . import stock_move
from . import stock_move_line
from . import stock_picking
```